assets/js/utils/api.js

```diff
@@ -12,59 +12,59 @@ const ACTIONS = {
  * @param {String} URL
  * @param {Any} payload
  */
-async function authenticatedRequest(action, URL, payload = null) {
+const authenticatedRequest = async function(action, URL, payload = null) {
   const config = { 'headers': { 'X-WP-Nonce': nonce } };
   return await axios({
     method: action,
     url: URL,
     data: payload,
-    ...config
+    ...config,
```

Comment:
please revert all the `,` changes please 😉 
without ending `,` look more pleasing to me. 😂